src/simba/feedback_survey/index.html

```
@@ -0,0 +1,87 @@
+---
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)